xsoar/resource_integration_instance.go

```diff
@@ -169,8 +172,15 @@ func (r resourceIntegrationInstance) Create(ctx context.Context, req tfsdk.Creat
 			break
 		}
 	}
-	var configs map[string]string
-	plan.Config.ElementsAs(ctx, &configs, true)
+	var configs map[string]any
+	err = json.Unmarshal([]byte(plan.Config.Value), &configs)
+    if err != nil {
+		resp.Diagnostics.AddError(
+			"Error creating integration instance",
+			"Could not parse integration instance config json: "+err.Error(),
+		)
+		return
+    }
```

Comment:
The start and the end of the `if` statement are indented using spaces, while the rest of the file uses tabs. A quick format should resolve this :)